src/NLog/Targets/FileArchiveModes/FileArchiveModeBase.cs

```diff
@@ -111,7 +112,7 @@ public virtual List<DateAndSequenceArchive> GetExistingArchiveFiles(string archi
             }
 
             if (existingArchiveFiles.Count > 1)
-                existingArchiveFiles.Sort(FileSortOrderComparison);
+                existingArchiveFiles.Sort((x,y) => FileSortOrderComparison(x, y));
```

Comment:
Why this change? Do you think it's more clear then?

Reply:
Just removes delegate allocation :)

Reply:
I really was under the impression that the "method group" call was better I'm perf (no anonymous method) , but have found a good article on that. Do you have something to read for me?